cassandra/cqlengine/connection.py

```diff
@@ -28,67 +28,200 @@
 
 NOT_SET = _NOT_SET  # required for passing timeout to Session.execute
 
-Host = namedtuple('Host', ['name', 'port'])
-
-cluster = None
```

Comment:
I would suggest do it for 3.x, but remove in 4.0. So cluster will always point to the DEFAULT cluster object.